### PR TITLE
[ADD] social marketing: added social marketing documentation to v14

### PR DESCRIPTION
**Pull request by ksc-odoo**
Task ID: [2850055](https://www.odoo.com/web#id=2850055&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hi @mivu-odoo 

here's the social marketing doc for v14. If you could please review it at your earliest convenience, that would be greatly appreciated.

Once you approve, I'll be sure to send it along to Zac for further technical review.

Thanks!

**Comment by mivu-odoo** [comment]
As of commit [991c8dc](https://github.com/odoo/documentation/commit/991c8dcdd76065c92817e5d4c408e0a7677b575a)

Fixed the build errors! This PR passes all the code checks now.

- Removed unexpected indent in `create_posts.rst`
- Fixed file path reference in `campaigns.rst` and `push_notifications.rst`, it was referencing a file called `manage_social` but that file didn't exist when I searched the repo. I'm guessing that was the old name for `create_posts.rst` so I updated the file path references to `create_posts`

**Comment by ksc-odoo** [comment]
@StraubCreative Ready for your review!

**Comment by StraubCreative** [comment]
@ksc-odoo please update this PR in reference to change requests left on other Marketing PRs as well as the content guidelines. Tag @samueljlieber for help with RST and then tag me again when ready 🙂

**Review by StraubCreative — CHANGES_REQUESTED** [review]
@ksc-odoo please update this PR in reference to change requests left on other Marketing PRs as well as the content guidelines. Tag @samueljlieber for help with RST and then tag me again when ready 🙂

**Comment by samueljlieber** [comment]
Hey @ksc-odoo 👋 in https://github.com/odoo/documentation/commit/db369925b6da89f021e19f0d4f40af3563a7b98c I updated the RST content with what you sent over, great job on this BTW!

I did make some structural changes to follow the same structure as the SMS Marketing docs (#2553):
- instead of having your `essentials.rst` & `campaigns.rst` in an *overview* folder, I changed *overview* folder to *essentials*, and moved `social_essentials.rst` & `social_campaigns.rst` into that folder.
    - @ksc-odoo what do you think, should we keep the previous *overview* folder structure?
- removed `push_notifications.rst` & `livechat.rst` to consolidate into `social_essentials.rst` & `social_campaigns.rst`
- added redirects for deleted files

**Comment by samueljlieber** [comment]
Fixed merge conflict in https://github.com/odoo/documentation/commit/e369e35a343a6eda29c8d4c8c509ec3d5b2b2499.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
@ksc-odoo can you give these another pass around the following:

- use of "quotes" and (parenthesis), often times these aren't needed / are distracting.
- remove all hyperbole/exaggeration language (e.g. _utmost importance_)
- specificity of language (e.g. _social marketing campaigns help companies connect with the marketplace in uniquely creative
ways._)
- wordiness...some spots where you can say things more simply/direct and stick to active instruction

Think these just need a final round of love and should be good for merge. Thank you!

File: content/applications/marketing/social_marketing/essentials/social_campaigns.rst
Comment (ksc-odoo) [reply content/applications/marketing/social_marketing/essentials/social_campaigns.rst:91]: For sure. I believe the majority of this doc was done awhile back, so there were a few more issues than usual, but I really appreciate you pointing them out/fixing them!

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (ksc-odoo) [reply content/applications/marketing/social_marketing/essentials/social_essentials.rst:282]: Great call!

File: content/applications/marketing/social_marketing/essentials/social_campaigns.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_campaigns.rst:10]: Suggestion: wording/specificity 

```suggestion
Social marketing campaigns help companies connect directly with the marketplace. These campaigns
are helpful when introducing a new product to customers, explaining the value of a product or service,
or when advertising an upcoming event or product release.

The most effective social marketing campaigns typically involve multiple channels to maximize
content distribution, and Odoo's *Social Marketing* application acts as a singular control center to
monitor, plan, post, track, and analyze all of the various content and content channels within a
single dashboard.
```

File: content/applications/marketing/social_marketing/essentials/social_campaigns.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_campaigns.rst:27]: I know there's some uncertainty around guilabel use and icons.
Imo it makes more sense to use them because we're instructing or hinting at the user to click them.
Ideally, I'd love to use emojis here (maybe one day).

Here's what I suggest, with that in mind, and also turning the second sentence into a **note**:
```suggestion
Each *stage* in the kanban view can be edited, by clicking the :guilabel:`gear icon` to the left of
the :guilabel:` + (plus sign icon)` - located to the right of the stage title.

.. note::
   The **gear icon** *only* appears when the cursor hovers to the left of the ** + (plus sign
   icon)**. When the gear icon is clicked, a drop-down menu reveals the options: :guilabel:`Fold`,
   :guilabel:`Edit Stage`, and :guilabel:`Delete`.
```

File: content/applications/marketing/social_marketing/essentials/social_campaigns.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_campaigns.rst:39]: Since we're actively on this dashboard and providing a UI walk-through, these get guilabels.

```suggestion
Clicking :guilabel:`Fold` minimizes that specific stage's column. The stage column can be restored
by clicking the folded version of it on the main :guilabel:`Campaigns` dashboard in the default
kanban view.

Selecting :guilabel:`Edit Stage` reveals a pop-up window, in which the name and the sequence of the
stage can be modified. If changes are made, be sure to click :guilabel:`Save`.

Clicking :guilabel:`Delete` removes the stage entirely.
```

cc: @samueljlieber

File: content/applications/marketing/social_marketing/essentials/social_campaigns.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_campaigns.rst:43]: **Campaigns** is the dashboard heading that we're  currently on, so it gets a guilabel.

```suggestion
.. note::
   To add a new stage to the pipeline, side-scroll to the right on the :guilabel:`Campaigns`
   dashboard, click :guilabel:`Add a Column`, enter in the desired information, and click
   :guilabel:`Add`.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
@samueljlieber stopping here.

Please clean up the formatting on these docs:

- italics where there should be guilabels
- missing guilabels around icons (gear, +, etc.) @ksc-odoo I know you were told different  but it's okay if there's disagreement. Imo these are clickable UI objects so they deserve them.
- italics where there shouldn't be any
- missing guilabels on dashboard headings
- capitalizations where there shouldn't be any (e.g. Search bar --> search bar)

Really the only change we've introduced lately is to stop using guilabels on _everything_, and to only limit it to context around active instruction. I get that there's some confusion and it'll take a few tries (and that's totally fine).

To define this better: active instruction still fulfills a majority of cases, and _includes_ UI walk-throughs where we're looking around the dashboard (because we're still talking about the dashboard that's currently in view).

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:99]: guilabel vs. italics use
```suggestion
.. important::
   In order for the :guilabel:`Push Notification` option to appear, make sure the *Enable Web Push
   Notifications* feature is enabled in the *Website* app. To do that, navigate to
   :menuselection:`Website --> Configuration --> Settings`, activate :guilabel:`Enable Web Push
   Notifications`, fill out the corresponding fields, and click :guilabel:`Save`.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:117]: Good block. Would just guilabel the icon 😉 
```suggestion
.. tip::
   Emojis can also be added directly to the text in the :guilabel:`Message` field. Just click the
   :guilabel:`emoji (smiley face) icon`, located on the line of the :guilabel:`Message` field to the
   far right. Clicking this icon reveals a drop-down containing numerous emojis to choose from.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:133]: Remove colloquialism.
Add slightly more specificity around the type of menu (i.e., field vs main vs. header menu).
```suggestion
.. tip::
   A new campaign can be created, as well, by typing the name of the new campaign on the blank
   :guilabel:`Campaign` field, and selecting :guilabel:`Create` from the drop-down field menu. Or,
   select :guilabel:`Create and edit` from the menu to further customize that newly-created
   campaign.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:150]:
```suggestion
time, by clicking the :guilabel:`scheduling (clock) icon` located on the calendar pop-up, and choose
the desired time for Odoo to publish this post on that future date.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:155]: The last part of the sentence mixes up the subject(s) of the sentence, so we can remove it.

```suggestion
the post to :guilabel:`Scheduled`.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:184]: guilabel formatting
```suggestion
To upload a new image, click the :guilabel:`Edit (pencil) icon` when the :guilabel:`Push Icon Image`
field is hovered over with the cursor. Then, proceed to locate and upload the desired image, and
Odoo automatically displays a preview of how the icon will appear on the push notification.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:216]: Apostrophes aren't needed?
```suggestion
Posts page
==========
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:270]:
```suggestion
      Click on any :guilabel:`+ (plus sign) icon` next to a line in the pivot table to reveal more metric
      options to add to the grid.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:284]:
```suggestion
      - :guilabel:`Flip Axis`, which switches the *X* and *Y* axis in the grid table.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:294]: Force the tag
```suggestion
database, navigate to :menuselection:`Social Marketing --> Visitors` in the header menu.
```

File: content/applications/marketing/social_marketing/essentials/social_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/social_marketing/essentials/social_essentials.rst:341]: Grammar
```suggestion
Navigate to :menuselection:`Configuration --> Social Streams` to reveal a separate page containing
all of the social media streams that have been added to the main dashboard of the *Social Marketing*
app, accessible via the :guilabel:`Feed` option in the header menu.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

Made a batch of edits in the commit following this review, mostly around guilabel/italics use (I know it's annoying/confusing!), and some wording and formatting suggestions.

Please let me know if you approve of the changes or If you have any disagreements, thanks!

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
